d3a_api_client/enums.py

```diff
@@ -9,16 +9,26 @@ class Commands(Enum):
     LIST_OFFERS = 5
     LIST_BIDS = 6
     DEVICE_INFO = 7
+    UPDATE_OFFER = 8
+    UPDATE_BID = 9
+    GRID_FEES = 10
+    MARKET_STATS = 11
+    DSO_MARKET_STATS = 12
 
 
 command_enum_to_command_name_dict = {
     Commands.OFFER: "offer",
+    Commands.UPDATE_OFFER: "update_offer",
     Commands.BID: "bid",
+    Commands.UPDATE_BID: "update_bid",
     Commands.DELETE_OFFER: "delete_offer",
     Commands.DELETE_BID: "delete_bid",
     Commands.LIST_OFFERS: "list_offers",
     Commands.LIST_BIDS: "list_bids",
-    Commands.DEVICE_INFO: "device_info"
+    Commands.DEVICE_INFO: "device_info",
+    Commands.GRID_FEES: "grid_fees",
+    Commands.MARKET_STATS: "market_stats",
+    Commands.DSO_MARKET_STATS: "dso_market_stats"
```

Comment:
I thought we decided to deprecate these. I think this was only discussed in PRs and only mentioned in planning once. Shall I create a ticket for that @spyrostz ?

Reply:
@hannesdiedrich Yes I am aware that we are to deprecate these , but this was not in the context of this ticket

Reply:
Please create a ticket for that, these are probably not going to be needed again.

Reply:
https://gridsingularity.atlassian.net/browse/D3ASIM-3351